sigle/src/modules/settings/plans/CurrentPlan.tsx

```diff
@@ -198,6 +215,19 @@ export const CurrentPlan = () => {
         </>
       ) : null}
 
+      {!isLoading && isError && (
```

Comment:
Wdyt about using the isLegacy variable injected by useAuth? isError can come from a network failure for example

Reply:
yep that makes sense! will update it 👍